src/components/filter/filter-store.tsx

```diff
@@ -70,10 +45,6 @@ const zoomUrlKey = "zoom";
 const latUrlKey = "lat";
 const lngUrlKey = "lng";
 
-const ageIntervalSearch = new URL(window.location.href).searchParams.getAll(
-	treeAgeUrlKey,
-);
-
```

Comment:
we still need something to read the state of the URL treeRange query param on initial page view